src/core/settings.py

```diff
@@ -241,8 +243,9 @@ def _load_api_raml(path):
 
 INGESTION_LOG_NAME = 'ingestion-%s.log' % PROJECT_NAME
 
-LOG_FILE = join('/var/log/', LOG_NAME) # ll: /var/log/lax.log
-INGESTION_LOG_FILE = join('/var/log/', INGESTION_LOG_NAME) # ll: /var/log/lax.log
+LOG_DIR = '/var/log/' if ENV != DEV else './'
```

Comment:
https://github.com/elifesciences/lax/commit/edf8bde3c9a9674728b2445c1b52203fa5414f4f essentially the uwsgi-lax process couldn't write file like `/srx/lax/lax.log` in the Vagrant VM; how do we make it work both in the VM and in local checkouts of Lax?